great_expectations/expectations/core/schemas/ExpectColumnDistinctValuesToBeInSet.json

```diff
@@ -280,17 +285,6 @@
                 "offset"
             ],
             "additionalProperties": false
-        },
-        "ConditionParser": {
-            "title": "ConditionParser",
-            "description": "Type of parser to be used to interpret a Row Condition.",
-            "enum": [
-                "great_expectations",
-                "great_expectations__experimental__",
-                "pandas",
-                "spark"
```

Comment:
The strings `great_expectations__experimental__` and `spark` will continue to work at runtime, but they will raise type errors with static analysis tools.